### ISSUE TITLE
Commit hash is not created when calling the install scipts

### ISSUE THREAD

**Issue by HenrikJannsen** [bisq-network/bisq2#2616]
Using `./gradlew :apps:oracle-node-app:installDist` or `./gradlew :apps:seed-node-app:installDist` does not create or update the commit hash

**Comment by HenrikJannsen** [comment]
@rodvar Could you have a look?

**Comment by HenrikJannsen** [comment]
I looked closer into it. The problem is that the common project does not get built if there was no change in the sources. 
We need to call a clean first. As common is a low level dependency it will rebuild the whole project. We don't want to have that by default as it would slow down builds... So I fear we just need to ensure that in the task where its important we need to ensure clean is called. Those are the generateInstaller task and the installDist tasks

**Comment by rodvar** [comment]
> I looked closer into it. The problem is that the common project does not get built if there was no change in the sources. We need to call a clean first. As common is a low level dependency it will rebuild the whole project. We don't want to have that by default as it would slow down builds... So I fear we just need to ensure that in the task where its important we need to ensure clean is called. Those are the generateInstaller task and the installDist tasks

For sure @HenrikJannsen , I'll have a look into this.

Just thinking out loud now, maybe is better to have a script that does the right steps from the root instead of tweaking each individual installDist and generateInstaller tasks?

It would be a script for the people in charge of generating the builds, cause day-to-day dev won't need it.

in your example above, it would. be something like

`
#!/bin/bash
# generateInstallers.sh
./gradlew clean build
./gradlew :apps:oracle-node-app:installDist
./gradlew :apps:seed-node-app:installDist
# other installDist...
`

Let's decide first the best way to tackle this and then I'll just implement it, what are your thoughts?

**Comment by HenrikJannsen** [comment]
Yes a clean does the job. We can leave it as it seems not that easy to include it in installDist.
